RetroFE/Source/Graphics/Page.cpp

```diff
@@ -54,7 +54,7 @@ Page::Page(Configuration &config, int layoutWidth, int layoutHeight)
         layoutWidth_.push_back(layoutWidth);
         layoutHeight_.push_back(layoutHeight);
     }
-    for (int i = 0; i < SDL::getNumScreens(); i++)
+    for (int i = 0; i < SDL::getScreenCount(); i++)
```

Comment:
Yup I agree with these changes since

    screenCount_ = std::min(numScreens_, numDisplays_);

Which supports user selection of screens vs max number of possibly screens